iocage_lib/ioc_common.py

```diff
@@ -801,9 +787,12 @@ def runscript(script):
     else:
         return False, 'Script is not executable!'
 
-    success, output = safe_checkoutput(script, stderr=su.STDOUT)
+    with iocage_lib.ioc_exec.IOCExec(
+        script, None, None, unjailed=True, decode=True
+    ) as _exec:
+        success, error = list(_exec)[0]
```

Comment:
Perhaps we extend SilentExec to actually return stdout, stderr instead of just passing and using a list on the generator here?